docs/source/manual/configuration.rst

```diff
@@ -1313,6 +1314,7 @@ Name                         Default            Description
 protocol                     TLSv1.2            The default protocol the client will attempt to use during the SSL Handshake.
                                                 See
                                                 `here <http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#SSLContext>`_ for more information.
+provider                     (none)             The name of the JCE provider to use on client side for cryptographic support.
```

Comment:
Could you please add some examples for valid values here, e. g. "BC" for [BouncyCastle](https://www.bouncycastle.org/) or [Conscrypt](https://github.com/google/conscrypt)?